pkg/shared/families/malware/yara/util/util.go

```diff
@@ -43,6 +44,18 @@ func (e *InvalidLineError) Error() string {
 	return fmt.Sprintf("invalid yara line: %s", e.Line)
 }
 
+func IsErrorThresholdReached(successCount, allCount int) bool {
+	switch allCount {
+	case 0:
+	default:
+		if float64(successCount)/float64(allCount) < errThreshold {
+			return true
+		}
+	}
```

Comment:
Using a switch here is a really weird to me I think this can be:

```
if allCount != 0 && float64(successCount)/float64(allCount) < errThreshold {
   return true
}
return false
```

I think its a little confusing that errThreshold is called errThreshold when we're actually comparing it to number of successes. Perhaps it would be easier to understand if the number of errors was the input and we did something like:

```
errThreshold = 0.8 
if float64(errorCount) / float64(allCount) > errThreshold
```